libraries/testing/include/eosio/testing/tester.hpp

```diff
@@ -390,7 +390,7 @@ namespace eosio { namespace testing {
             return genesis;
          }
 
-         static std::pair<controller::config, genesis_state> default_config(const fc::temp_directory& tempdir) {
+         static std::pair<controller::config, genesis_state> default_config(const fc::temp_directory& tempdir, optional<uint32_t> genesis_max_inline_action_size = optional<uint32_t>{}, optional<uint32_t> genesis_max_nonprivileged_inline_action_size = optional<uint32_t>{}) {
             controller::config cfg;
```

Comment:
`genesis_max_nonprivileged_inline_action_size` was renamed to `config_max_nonprivileged_inline_action_size` above but not here or below.